file_actions.py

```diff
@@ -10,11 +11,11 @@ def open_file(self):
         file = open(self.file_name, "w+")
         self.file = file
 
```

Comment:
pep8: IndentationError: unindent does not match any outer indentation level. Code: E901